src/documentation/domains/issue/controller/issue.controller.docs.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     TimeTrackingEvent:
+ *       type: object
+ *       properties:
+ *         id:
+ *           type: string
+ *           description: The ID of the event.
+ *         startTime:
+ *           type: string
+ *           format: date-time
+ *           description: The start time of the event.
+ *         endTime:
+ *           type: string
+ *           format: date-time
+ *           description: The end time of the event.
+ *         issueId:
+ *           type: string
+ *           description: The ID of the issue associated with the event.
+ *   parameters:
+ *     IssueIdParam:
+ *       in: path
+ *       name: id
+ *       required: true
+ *       schema:
+ *         type: string
+ *       description: The ID of the issue to pause the timer for.
+ * paths:
+ *   /api/issue/{id}/pause:
+ *     get:
+ *       summary: Pause Timer for an Issue
+ *       tags:
+ *         - Issue
+ *       description: Pause the timer for a specific issue.
+ *       parameters:
+ *         - $ref: '#/components/parameters/IssueIdParam'
+ *       responses:
+ *         '200':
+ *           description: "Successful operation"
+ *           content:
+ *             application/json:
+ *               schema:
+ *                 $ref: '#/components/schemas/TimeTrackingEvent'
```

Comment:
For some reason swagger docs are showing an example of error 401 and not 400. Check how reference are done in other endpoints so it is shown properly

Reply:
should add error 401 too

Reply:
Solved by splitting docs in different files.